packages/deployer/src/cli/commands.ts

```diff
@@ -21,13 +21,12 @@ export async function deploy(configFile: string, secretsFile: string, receiptFil
 
   const mnemonic = config.nodeSettings.airnodeWalletMnemonic;
   if (!validateMnemonic(mnemonic)) {
-    logger.fail('AIRNODE_WALLET_MNEMONIC in your secrets.env file is not valid');
-    throw new Error('Invalid mnemonic');
+    throw new Error('AIRNODE_WALLET_MNEMONIC invalid or not found');
   }
 
   const httpGateway = config.nodeSettings.httpGateway;
   let httpGatewayApiKey: string | undefined = undefined;
-  if (httpGateway.enabled) {
+  if (httpGateway?.enabled) {
```

Comment:
This is not correct. The HTTP gateway and heartbeat are optional features, but you need to explicitely turn them off and this property should be present. We should update the docs if this is not clear.

The schema check for this should be handled in https://api3dao.atlassian.net/browse/AN-284

Reply:
That's because you are looking at `pre-alpha` docs. It's correct in https://docs.api3.org/airnode/next/grp-providers/tutorial/config-json.html